templates/kubevirt/vm-route-forge/daemonset.yaml

```diff
@@ -31,34 +31,34 @@ spec:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
```

Comment:
There is no health probe in a DaemonSet manifest.

Reply:
will be solved in another task